packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -91,17 +107,21 @@ class MediaQueryData {
   ///    textScaleFactor defined for a [BuildContext].
   final double textScaleFactor;
 
-  /// The number of physical pixels on each side of the display rectangle into
-  /// which the application can render, but over which the operating system
-  /// will likely place system UI, such as the keyboard, that fully obscures
-  /// any content.
+  /// The parts of the display that are completely obscured by system UI,
+  /// typically by the device's keyboard.
+  ///
+  /// When a mobile device's keyboard is visible `viewInsets.bottom`
```

Comment:
Bah, sorry I might be totally wrong on this. But `When a mobile device's keyboard is visible, viewInsets.bottom` seems more natural. Anyway, it's aesthetics. Feel free to ignore.

Reply:
Because of all of the comma comments I just read a grammer site's overview of the rules for inserting commas. For as long as I could stand it anyway. As far as I can tell commas like this, which would match the natural pauses in speech, don't belong in written text.

Reply:
nit: `Because of all of the comma comments, I just read a grammer site's overview of the rules`

I kid I kid. Like I said, you have a better grasp at this than I would.